### PR TITLE
added navigation

### PR DESCRIPTION
**Pull request by MelGGit**
Added navigation for my page

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,29 +1,34 @@
...
-  function handleClickFavoritesPage() {
-    push('/favorites')
-  }
 }
 
-export default App
+const AppContainer = styled.section`
+  height: 100vh;
+  display: grid;
+  grid-template-rows: 1fr 48px;
```
Comment (JGieseWeb) [inline src/App.js:33]: Maybe use relative size 48px = 3em :)

**Review by badabam — APPROVED** [review]
I really like what you do. First time I see the upcoming state library `recoil` in a project. Nice!

File: src/components/PodcastCard/PodcastCard.spec.js [hunk 2]
```diff
@@ -1,20 +1,25 @@
...
-import userEvent from '@testing-library/user-event'
 import PodcastCard from './PodcastCard'
 // needed since the component PodcastCard is lazy, it doesnt show in the jsdom
 // therefore the function forceVisible() is to show it in the dom for testing
 import { forceVisible } from 'react-lazyload'
+import { Router } from 'react-router-dom'
 
 describe('PodcastCard', () => {
   it('renders an image and title', () => {
+    const historyMock = { push: jest.fn(), location: {}, listen: jest.fn() }
```
[the remark this answers is not in the document]
Comment (MelGGit) [reply src/components/PodcastCard/PodcastCard.spec.js:10]: Thank you, i changed it and hopefully did it the right way.

File: src/App.js (hunk 1 above)
Comment (MelGGit) [reply src/App.js:31]: Thank you for your review. Updated accordingly.

**Pull request merged**